src/extensions/core/maskeditor.ts

```diff
@@ -818,7 +887,16 @@ class MaskEditorDialog extends ComfyDialog {
           self.maskCtx.beginPath()
           self.maskCtx.fillStyle = this.getMaskFillStyle()
           self.maskCtx.globalCompositeOperation = 'source-over'
-          self.maskCtx.arc(x, y, brush_size, 0, Math.PI * 2, false)
+          if (self.pointer_type === 'rect') {
```

Comment:
Extract this logic into a function, as it is repeated over 3 times.

Reply:
this is corrected